perun-base/src/main/resources/perun-roles.yml

```diff
@@ -1059,7 +1071,13 @@ perun_policies:
       - MFA: Facility
       - MFA: User
```

Comment:
```suggestion
```
I know this isn't part of your PR, but I think we should omit User here, since copyManagers uses the setRole() function (for which we will check MFA separately IIRC)

Reply:
good point, fixed